Shared/mods/deathmatch/logic/luadefs/CLuaXMLDefs.cpp

```diff
@@ -185,11 +184,12 @@ int CLuaXMLDefs::xmlLoadFile ( lua_State* luaVM )
                             }
                         }
 
-                        xmlFile->GetLastError ( strError );
-                        const int exists = strstr ( strError, "Failed to open file" ) != NULL;
-                        if( exists==0 )
-                            argStream.SetCustomError ( strError, SString("Unable to read XML file %s", strFileInput.c_str() ) );
-
+                        if ( FileExists ( strPath ) ) {
```

Comment:
Put the **{** on a new line to match the code style and it should be merge-able.

Reply:
done